esp/esp/program/modules/handlers/teachercheckinmodule.py

```diff
@@ -169,8 +168,11 @@ def ajaxteachertext(self, request, tl, one, two, module, extra, prog):
                 sec = ClassSection.objects.get(id=request.POST['section'])
                 teacher = PersistentQueryFilter.create_from_Q(ESPUser, Q(username=request.POST['username']))
                 message = "Don't forget to check-in for your " + one + " class that is scheduled for " + sec.friendly_times(include_date = True)[0] + "!"
-                GroupTextModule.sendMessages(teacher, message, True)
-                return {'message': "Texted teacher"}
+                log = GroupTextModule.sendMessages(teacher, message, True)
+                if "error" in log:
```

Comment:
Will this cause problems if the teacher's name contains "error", or something?  It's a little hacky but maybe fine.